### ISSUE TITLE
Airmon-ng error message with rt73

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#509]
**Reported by misterx on 28 Sep 2008 18:59 UTC**

Using rt73 3.0.1 from ASPj, here is the output of airmon-ng (aircrack-ng/aircrack-ng@f8b712dc3990e4d92bad8e26d0c552956917f1a2) when running "airmong-ng start rausb0 6":

```
rausb0 Ralink 2573 rt73/usr/local/sbin/airmon-ng: 
line 33: [: -lt: unary operator expected (monitor mode enabled)rausb0 no private ioctls.
```

Note that it was working with ~ aircrack-ng/aircrack-ng@

**Comment by aircrack-ng** [comment]
**Comment by misterx on 28 Sep 2008 18:59 UTC**

See also #505 for a fix.

**Comment by aircrack-ng** [comment]
**Modified by misterx on 28 Sep 2008 18:59 UTC**

**Comment by aircrack-ng** [comment]
**Modified by darkAudax on 28 Sep 2008 18:59 UTC**

**Comment by aircrack-ng** [comment]
**Comment by darkAudax on 28 Sep 2008 18:59 UTC**

Fixed in aircrack-ng/aircrack-ng@199cf836918d1e9245cb363f9b9303fa63bf0397.
